types/mousetrap/plugins/pause/mousetrap-pause.d.ts

```diff
@@ -1,4 +1,4 @@
-import { MousetrapStatic, MousetrapInstance } from "../";
+import { MousetrapStatic, MousetrapInstance } from "../..";
```

Comment:
```suggestion
import { MousetrapStatic, MousetrapInstance } from "../../";
```
I think it'd be better to keep the consistency.